docs/about/authors.rst

```diff
@@ -57,3 +57,4 @@ The following developers have contributed to the PyMeasure package:
 | Asaf Yagoda
 | Fabio Garzetti
 | Daniel Schmeer
+| David Sanchez Sanchez
```

Comment:
Since the last merge, we do not need the name in "authors.rst" anymore.
Just revert this change and the "merge conflict" message should disappear.

Reply:
Done. Thanks